api/Navigator.json

```diff
@@ -53,11 +53,11 @@
           "support": {
             "chrome": {
               "version_added": false,
-              "notes": "Available on all platforms behind a flag, but currently only works on desktop in an <a href='https://webvr.info/get-chrome/'>experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked)."
+              "notes": "Available on all platforms behind a flag, but currently only works on desktop in an <a href='https://webvr.info/get-chrome/'>experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked). WebVR support is no longer available in Chrome 80 and later."
```

Comment:
I'd propose removing this note entirely. It's contradicted by `"version_added": false` and we're not keeping non-standard build information anyway

Reply:
Agreed.